README.md

```diff
@@ -1,23 +1,54 @@
 # FHIR Works on AWS deployment
 
-FHIR Works on AWS is a framework that can be used to deploy a [FHIR server](https://www.hl7.org/fhir/overview.html) on AWS. Using this framework, you can customize and add different FHIR functionality to best serve your use cases. When deploying this framework, by default [Cognito and role based access control](https://github.com/awslabs/fhir-works-on-aws-authz-rbac) is used. However, if preferred, you can be authenticated and authorized to access the FHIR server’s resources by using [SMART](https://github.com/awslabs/fhir-works-on-aws-authz-smart) instead of Cognito. Cognito is the default AuthN/AuthZ provider because it is easier to configure than SMART. It doesn’t require setting up a separate IDP server outside of AWS as compared to SMART. However, Cognito authentication is not granular. When a new user is created, it is assigned into the auditor, practitioner, or non-practitioner groups. Depending on the group, the user gets access to different groups of FHIR resources.
-The AuthN/Z providers are defined in `package.json` and `config.ts`. You can choose  appropriate providers. SMART allows greater granularity into authentication than Cognito and is the FHIR standard. It allows you to access a FHIR record only if that record has reference to the user.
-
+FHIR Works on AWS is a software toolkit that can be used to deploy a [FHIR server](https://www.hl7.org/fhir/overview.html) on AWS. Using this toolkit, you can customize and add different FHIR functionality to best serve your use cases.
 
 ## FHIR Works on AWS features
 
-FHIR Works on AWS utilizes AWS Lambda, Amazon DynamoDB, Amazon S3 and Amazon Elasticsearch Service to provide the following FHIR features:
+FHIR Works on AWS implements the FHIR specification (STU3 or R4) and deploys that implementation into an AWS account using a set of mostly serverless AWS services.  It provides the following FHIR features:
+
++ Create, Read, Update, Delete (CRUD) operations for *all* R4 or STU3 base FHIR resources
++ Search by FHIR resource type
++ Versioned reads ([vread](https://www.hl7.org/fhir/http.html#vread))
++ Transaction bundle POSTs of up to 25 entries
++ Third-party Implementation Guides for localization
++ Authentication and authorization using either Role-Based Access Control or SMART
+
+## Architecture
+
+The system architecture comprises Amazon API Gateway, AWS Lambda, Amazon DynamoDB, Amazon S3, Amazon Elasticsearch Service, and optionally Amazon Cognito, plus CloudWatch Logs. The endpoint is published through API Gateway. The persistence layer (database and storage) uses Amazon DynamoDB and S3, with Elasticsearch as the search index for the data initially written to DynamoDB. The endpoint is secured by API keys, and by Cognito for user-level authentication and user-group authorization. CloudWatch Logs capture system events.  The diagram below shows the toolkit's architecture components and how they are related.
```

Comment:
The first use and the subsequent use of AWS services is provided here: https://w.amazon.com/bin/view/AWSDocs/editing/service-names

1. For DynamoDB, please change the subsequent use to DynamoDB in the third sentence.
2. For S3, the first use is Amazon Simple Storage Service (Amazon S3) and the subsequent use is Amazon S3.
3. The Service names list (https://w.amazon.com/bin/view/AWSDocs/editing/service-names) states that Elasticsearch is now called Opensearch service. Should we change it to Opensearch service?
4. Since this is the first occurence of CloudWatch in this para, as per the Service list, can we change it to "Amazon CloudWatch Logs"? Thereafter, we can use "CloudWatch Logs".